src/functions/retryProcessing/application/__tests__/CleanupUploadQueueRecords.int.ts

```diff
@@ -114,56 +120,51 @@ describe('Clean up operations', () => {
         applicationReference: RetryUploadCleanUpTestCases.TarsAcceptedNotifyAcceptedRsisAccepted,
         staffNumber: '1',
         timestamp: '2019-09-01 13:59:59',
-        interface: 0, // TARS
-        uploadStatus: 1, // ACCEPTED
+        interface: InterfaceIds.TARS, // TARS
+        uploadStatus: ProcessingStatus.ACCEPTED, // ACCEPTED
         retryCount: 0,
       },
       {
         applicationReference: RetryUploadCleanUpTestCases.TarsAcceptedNotifyAcceptedRsisAccepted,
         staffNumber: '1',
         timestamp: '2019-09-01 13:59:59',
-        interface: 1, // NOTIFY
-        uploadStatus: 1, // ACCEPTED
+        interface: InterfaceIds.NOTIFY, // NOTIFY
+        uploadStatus: ProcessingStatus.ACCEPTED, // ACCEPTED
```

Comment:
Now you are using enmus the comments seem a bit pointless

Reply:
Definitely - removed them in other files. will remove these.